dataplane/src/config.rs

```diff
@@ -0,0 +1,20 @@
+use std::collections::HashMap;
+
+pub struct Vpc {
+    #[allow(dead_code)]
+    pub vni: u32,
+}
```

Comment:
I've created https://github.com/githedgehog/dataplane/pull/225 to move the `struct Vpc` from the `nat` module into a separate, dedicated module so that it would be easier to pull it here.

Reply:
Yep, we should use that when ready, but config is just a dummy.